docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+database:
+  image: "sameersbn/mysql:latest"
+  environment:
+    DB_NAME: staytus
+    DB_USER: staytus
+    DB_PASS: staytus
+  volumes:
+    - "/opt/docker/staytus-database:/var/lib/mysql"
+staytus:
+  #image: "adamcooke/staytus:latest"
+  image: "staytus-test"
```

Comment:
Oops, yes you are right the outcommented is for the end users. I'm gonna fix this in a moment.